csharp_dotnetcore/21.Custom-Dialogs/BotAccessors.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+
+namespace Microsoft.BotBuilderSamples
+{
+    /// <summary>
+    /// The accessors we will be using in the bot logic. Having a class like this just allows them to be easily
+    /// handed to the IBot instance through the dependency injection.
```

Comment:
Consider improving the wording on this sentence.

Reply:
done